tka/tailchonk.go

```diff
@@ -700,6 +700,13 @@ func markAncestorIntersectionAUMs(storage Chonk, verdict map[AUMHash]retainState
 			if err != nil {
 				return AUMHash{}, fmt.Errorf("reading %v: %w", h, err)
 			}
+
+			// TODO(alexc): I don't understand what this error message means, and I was
+			// quite confused by it -- in particular, if you try to run a compact, but
+			// none of the AUMs are eligible for deletion yet, you can hit this error.
+			//
+			// Can we write a better error message here, or better describe what this
+			// scenario means?
```

Comment:
The error that follows is the reason why a failing TKA compaction is a log, rather than an error – the first time I wrote a test for this, I started hitting this error. I tried investigating, but I've spent enough time on tangents already.